etc/chaos_supervisor.conf

```diff
@@ -1,5 +1,5 @@
 [program:chaos]
-command=/root/.virtualenvs/chaos/bin/python chaos.py
+command=bash chaos_wrapper.sh
```

Comment:
That's a surprise. Perhaps it would work if the full path to the script was given?